ci/owncloud/owncloud_definitions.sh

```diff
@@ -105,6 +105,33 @@ browse_url() {
     echo "$browse_url"
 }
 
+download_url() {
+    # Calculate and echo the download url for a file
+    #
+    # $1 relative path to the file from the base_url
+    # $2 base_url
+    local download_url
+
+    download_url="$(browse_url "$(dirname "$1")" "$2"/download)&files=$(basename "$1")"
+    echo "$download_url"
+}
+
+download_artifact() {
+    # Downloads a file from the artifacts public folder
+    #
+    # $1 relative path from the artifacts folder
+    # $2 local path including filename
+    # Uses:
+    #   OWNCLOUD_PUBLIC_URLS["artifacts"]
+    local base_url download_path
+
+    base_url="${OWNCLOUD_PUBLIC_URLS["artifacts"]}"
+    download_url="$(download_url "$1" "$base_url")"
```

Comment:
So, with my earlier suggestion, this would become
```suggestion
    local download_path

    download_url="$(download_url "artifacts" "$1")"
```